modules/ossm-federation-config-import.adoc

```diff
@@ -77,7 +77,7 @@ spec:
   importRules:
   - type: NameSelector
     importAsLocal:
-|Set to `true` to aggregate remote endpoint with local services. When `true`, services will be imported as `<name>.<namespace>.svc.cluster.local`
+|Set to `true` to aggregate remote endpoint with local services. When `true` services will be imported as `<name>.<namespace>.svc.cluster.local`. When `true` an alias is required. When `false` no alias is required. 
```

Comment:
I'd just suggest trying to write the second sentence in present tense per the ISG's [guidance on verb tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense):
```suggestion
|Set to `true` to aggregate remote endpoint with local services. When `true` services are imported as `<name>.<namespace>.svc.cluster.local`. When `true` an alias is required. When `false` no alias is required. 
```

Reply:
Made this edit.